### DISCUSSION TITLE
I'm not receiving e-mails about my ticket on another account

### DISCUSSION THREAD

**Discussion by cronta** [community/community#112221] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi there! My name is William.
Recently, my original account was suspended.

I was tagged on a spam comment on an issue I didn't participate in.
I answered the comment with "spam", and since there were plenty of people tagged on that comment, I was flagged as a spammer.

I didn't receive any e-mail about it. Opened the #2756974 ticket, and can't open any other anymore.
It says that my ticket was updated weeks ago, but I didn't receive any e-mail, so I'm not able to participate with that ticket, since there's no way to get to it.

Please, can you help me? I'm the owner of the account and I can prove everything, from e-mail, to password, to 2FA.

Please, can you help me? It's been almost a MONTH since I don't have access to my account, and I need it with some urgency.
There's 2 business on that account, and I need my code.

Anything, please, just help me get my account back.
I created this account to try and recover mine, so please, don't ban me because of this.

**Comment by bkpaccountwilldeletelater** [comment]
Yeah, same here, since Feb 12th I'm waiting for an answer from support to get my 10+ year account back. It seems that if you do not pay for a premium account, they don't really care.

**Reply by AbdullahKaraosmanoglu1** [reply]
Hello my friend,
I'm trying to get my main account back and they haven't responded for a month. Did you solve your problem?
